modules/vaos/spec/request/appointments_request_spec.rb

```diff
@@ -193,6 +193,15 @@
         end
       end
 
+      context 'when the upstream service returns an http status of 204, no content' do
```

Comment:
We can return a 404 to the website, but VAMF is returning a 204 for some reason.

Reply:
I think we should open a JIRA ticket to see why they're sending us a 204, since that doesn't really make sense.

But for now, I think returning 404 from vets-api is the best solution. The front end would have to treat a successful response with no data as an error anyway.

Reply:
updated return value to vets website from 204 to 404